dali/kernels/imgproc/flip_cpu_test.cc

```diff
@@ -45,38 +45,40 @@ class FlipCpuTest
   int flip_x_;
   int flip_y_;
   int flip_z_;
-  TensorShape<4> shape_;
+  TensorShape<5> shape_;
   std::vector<float> data_;
-  OutTensorCPU<float, 4> in_view_;
+  OutTensorCPU<float, 5> in_view_;
 };
 
 TEST_P(FlipCpuTest, ImplTest) {
   std::vector<float> out_data(volume(shape_));
   detail::cpu::FlipImpl(
       out_data.data(), in_view_.data,
-      shape_[0], shape_[1], shape_[2], shape_[3],
+      shape_[0], shape_[1], shape_[2], shape_[3], shape_[4],
```

Comment:
I am wondering, is it possible to work with a shape or arbitrary size and a layout, so that we can get the relevant dimensions and set to 1 the rest of them (that is, inside the implementation)